pkgs/tools/wayland/wl-clipboard/default.nix

```diff
@@ -21,18 +23,24 @@ stdenv.mkDerivation rec {
   };
 
   strictDeps = true;
-  nativeBuildInputs = [ meson ninja pkg-config wayland-scanner ];
+  nativeBuildInputs = [ meson ninja pkg-config wayland-scanner makeWrapper ];
   buildInputs = [ wayland wayland-protocols ];
 
   mesonFlags = [
     "-Dfishcompletiondir=share/fish/vendor_completions.d"
   ];
 
+  # Fix for https://github.com/NixOS/nixpkgs/issues/251261
+  postInstall = ''
+    wrapProgram $out/bin/wl-copy \
+      --suffix PATH : ${xdg-utils}/bin/xdg-mime
+  '';
+
   meta = with lib; {
     homepage = "https://github.com/bugaevc/wl-clipboard";
     description = "Command-line copy/paste utilities for Wayland";
     license = licenses.gpl3Plus;
-    maintainers = with maintainers; [ dywedir ];
+    maintainers = with maintainers; [ dywedir kashw2];
```

Comment:
```suggestion
    maintainers = with maintainers; [ dywedir kashw2 ];
```

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/251474/commits/d4ef31fc1464d1f806aaa329b8c000e586867d9c